ruby/lib/minitest/queue/runner.rb

```diff
@@ -48,6 +49,7 @@ def run_command
           BuildStatusRecorder.new(build: queue.build),
           JUnitReporter.new,
           OrderReporter.new(path: 'log/test_order.log'),
+          Reporters::StatsdReporter.new,
```

Comment:
I think it would better to check if statsd is configured here, and only add the reporter if it is.

Additionally, the `Configuration` object should be responsible for fetching the value from env.